test/controllers/beneficial.owner.type.controller.spec.ts

```diff
@@ -37,13 +40,26 @@ describe("BENEFICIAL OWNER TYPE controller", () => {
   });
 
   describe("GET tests", () => {
-    test("renders the beneficial owner type page for beneficial owners with all options", async () => {
+    test("renders the beneficial owner type page for beneficial owners with trusts", async () => {
       mockGetApplicationData.mockReturnValueOnce(APPLICATION_DATA_MOCK);
       const resp = await request(app).get(config.BENEFICIAL_OWNER_TYPE_URL);
 
       expect(resp.status).toEqual(200);
       expect(resp.text).toContain(BENEFICIAL_OWNER_TYPE_PAGE_HEADING);
       expect(resp.text).toContain(config.BENEFICIAL_OWNER_STATEMENTS_URL); // back button
+      expect(resp.text).not.toContain(CHECK_YOUR_ANSWERS_LINK); // continue button
```

Comment:
Just checking this is right - both the 'check your answers' link and 'enter trust info' link should be present?